src/app/visit/treatment.cpp

```diff
@@ -0,0 +1,29 @@
+#include "treatment.hpp"
+
+#include <map>
+#include <stdexcept>
+
+namespace
+{
+const std::map<Treatment, std::string> DentalTreatments = {
+    {Treatment::TeethCleaning, "Teeth Cleaning"},   {Treatment::RootCanal, "Root Canal"},
+    {Treatment::DentalFilling, "Dental Filling"},   {Treatment::ToothExtraction, "Tooth Extraction"},
+    {Treatment::TeethWhitening, "Teeth Whitening"}, {Treatment::DentalImplants, "Dental Implants"},
+    {Treatment::Orthodontics, "Orthodontics"},      {Treatment::Periodontics, "Periodontics"},
+    {Treatment::DentalVeneers, "Dental Veneers"},   {Treatment::DentalCrowns, "Dental Crowns"},
+    {Treatment::DentalBridges, "Dental Bridges"},   {Treatment::Dentures, "Dentures"},
+    {Treatment::GumSurgery, "Gum Surgery"},         {Treatment::DentalSealants, "Dental Sealants"},
+    {Treatment::MouthGuards, "Mouth Guards"},       {Treatment::SleepApneaTreatment, "Sleep Apnea Treatment"}};
+} // namespace
```

Comment:
Maybe think about adding a default treatment or call it "Other" if the treatment isn't in our enum

Reply:
Added "OTHER" value to enum.